### ISSUE TITLE
Pixel polishing for the wavelength slider

### ISSUE THREAD

**Issue by samreid** [phetsims/wave-interference#396]
https://github.com/phetsims/wave-interference/issues/377 said:

### Wavelength panel
- [x] Top-align with scale bar
- [x] Left-align with laser
- [x] Format like Wavelength NumberControl in Bending Light -- Readout between tweakers above track
- [x] Increase width of slider track 50%
- [x] Match height of slider track to FrequencySlider on earlier screens
- [x] Resize panel to fit updated contents 

However, some of this work is blocked by #394, so I've split it into this new issue.

**Comment by samreid** [comment]
After changes in #394, I added this code 
```js
      model.onProperty.linkAttribute( incidentBeam, 'visible' );
      model.onProperty.linkAttribute( transmittedBeam, 'visible' );

      const wavelengthToColor = VisibleColor.wavelengthToColor;
      const wavelengthPanel = new WaveInterferencePanel(
        new NumberControl( wavelengthString, model.wavelengthProperty, new Range( 380, 780 ), {
          titleNodeOptions: {
            font: WaveInterferenceConstants.DEFAULT_FONT
          },
          numberDisplayOptions: {
            font: new PhetFont( 14 ),
            valuePattern: '{0} nm'
          },
          sliderOptions: {
            trackNode: new SpectrumTrack( model.wavelengthProperty, {
              minValue: 380,
              maxValue: 780,
              valueToColor: wavelengthToColor,
              size: new Dimension2( 160, 20 )
            } ),
            thumbNode: new SpectrumThumb( model.wavelengthProperty, {
              valueToColor: wavelengthToColor,
              width: 25,
              height: 25,
              cursorHeight: 20
            } )
          },
          layoutFunction: NumberControl.createLayoutFunction3( {
            // alignTitle: 'left'
          } )
        } ), _.extend( PANEL_OPTIONS, {
          left: laserPointerNode.left,
          top: apertureScaleIndicatorNode.top,
          xMargin: 6
        } ) );
```
and now it looks like this:

![image](https://user-images.githubusercontent.com/679486/58775826-baf05280-8584-11e9-8f0c-75fb8507559f.png)

I am hesitant to commit this until the API is finalized in scenery-phet, but I wanted to check that the pixel polishing is as-desired (from the screenshot).  @arouinfar can you please comment?

**Comment by arouinfar** [comment]
@samreid the layout is looking great! I would suggest that the tweakers are made a bit smaller so their height matches the height of the wavelength readout.

![image](https://user-images.githubusercontent.com/8419308/58832785-ef0f5600-860c-11e9-937a-6e7498524b0a.png)

**Comment by samreid** [comment]
I finished the prescribed updates, @arouinfar can you please review on phettest/master?

**Comment by arouinfar** [comment]
Look great! Thanks @samreid.
